### DISCUSSION TITLE
[Feature] Count the API tokens based on different access passwords

### DISCUSSION THREAD

**Discussion by yehao1991** [ChatGPTNextWeb/NextChat#3989] [Ideas]
We want to promote it for use across the entire company, but with different access passwords. Therefore, we would like to count chat TOKENs based on the different access passwords. Is this feasible to develop?

**Comment by sfshine** [comment]
确实应该有这样的需求，方便对每个token进行细粒度管控，反而是随机token感觉用处不大
